src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -426,8 +427,7 @@ public void preInit(FMLCommonSetupEvent evt) {
         //VanillaPacketSplitter.register();
     }
 
-    public void loadComplete(FMLLoadCompleteEvent event) {
-    }
+//    public void loadComplete(FMLLoadCompleteEvent event) {}
```

Comment:
If you do not plan on using `loadComplete(FMLLoadCompleteEvent)` upon finishing this PR, please remove this one.